core/cat/routes/memory/collections.py

```diff
@@ -40,19 +41,14 @@ async def wipe_collections(
     """Delete and create all collections"""
 
     vector_memory: VectorMemory = stray.memory.vectors
-    collections = list(vector_memory.collections.keys())
+    deleted_memories = utils.delete_collections(vector_memory)
```

Comment:
if you move the `delete_collection` in the `vector_memory_collection.py` like
```
    def wipe(self) -> bool:
        try:
            self.client.delete(collection_name=self.collection_name)
            return True
        except Exception as e:
            log.error(f"Error deleting collection {self.collection_name}: {e}")
            return False
```

you could use

```
to_return = {str(c): ccat.memory.vectors.collections[str(c)].wipe() for c in MemoryCollection}
```

Reply:
Thanks for the suggestion! I made slightly different thing because there is direct access to the memory from the stray